data-source/json/test_list_change_evaluates_sections_primary_person.json

```diff
@@ -0,0 +1,340 @@
+{
```

Comment:
I think we should just have one test schema for this with a primary person as that schema can be used for all the test scernarios.